tests/unit/test-class-wc-payment-gateway-wcpay-subscriptions.php

```diff
@@ -512,6 +512,29 @@ public function test_render_custom_payment_meta_input_invalid_value() {
 		$this->wcpay_gateway->render_custom_payment_meta_input( $subscription, 'field_id', 'invalid_value' );
 	}
 
+	public function test_render_custom_payment_meta_input_many_tokens() {
+		$subscription = WC_Helper_Order::create_order( self::USER_ID );
+		$tokens       = [];
+
+		// Create 11 tokens and make it available to the subscription.
+
+		for ( $i = 1; $i <= 11; $i++ ) {
+			$token = WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_' . $i, self::USER_ID );
+			array_push( $tokens, $token );
+		}
+
+		foreach ( $tokens as $token ) {
+			$subscription->add_payment_token( $token );
+		}
+
+		$this->expectOutputString(
+			'<select name="field_id" id="field_id"><option value="" selected disabled>Please select a payment method</option><option value="' . $tokens[0]->get_id() . '" >' . $tokens[0]->get_display_name() . '</option><option value="' . $tokens[1]->get_id() . '" >' . $tokens[1]->get_display_name() . '</option><option value="' . $tokens[2]->get_id() . '" >' . $tokens[2]->get_display_name() . '</option><option value="' . $tokens[3]->get_id() . '" >' . $tokens[3]->get_display_name() . '</option><option value="' . $tokens[4]->get_id() . '" >' . $tokens[4]->get_display_name() . '</option><option value="' . $tokens[5]->get_id() . '" >' . $tokens[5]->get_display_name() . '</option><option value="' . $tokens[6]->get_id() . '" >' . $tokens[6]->get_display_name() . '</option><option value="' . $tokens[7]->get_id() . '" >' . $tokens[7]->get_display_name() . '</option><option value="' . $tokens[8]->get_id() . '" >' . $tokens[8]->get_display_name() . '</option><option value="' . $tokens[9]->get_id() . '" >' . $tokens[9]->get_display_name() . '</option><option value="' . $tokens[10]->get_id() . '" >' . $tokens[10]->get_display_name() . '</option></select>'
```

Comment:
Nitpick: We should split this into multiple lines for ease of reading.

Reply:
Something like this:

```suggestion
	        '<select name="field_id" id="field_id">' .
            '<option value="" selected disabled>Please select a payment method</option>' .
            '<option value="' . $tokens[0]->get_id() . '" >' . $tokens[0]->get_display_name() . '</option>' .
            '<option value="' . $tokens[1]->get_id() . '" >' . $tokens[1]->get_display_name() . '</option>' .
            '<option value="' . $tokens[2]->get_id() . '" >' . $tokens[2]->get_display_name() . '</option>' .
            '<option value="' . $tokens[3]->get_id() . '" >' . $tokens[3]->get_display_name() . '</option>' .
            '<option value="' . $tokens[4]->get_id() . '" >' . $tokens[4]->get_display_name() . '</option>' .
            '<option value="' . $tokens[5]->get_id() . '" >' . $tokens[5]->get_display_name() . '</option>' .
            '<option value="' . $tokens[6]->get_id() . '" >' . $tokens[6]->get_display_name() . '</option>' .
            '<option value="' . $tokens[7]->get_id() . '" >' . $tokens[7]->get_display_name() . '</option>' .
            '<option value="' . $tokens[8]->get_id() . '" >' . $tokens[8]->get_display_name() . '</option>' .
            '<option value="' . $tokens[9]->get_id() . '" >' . $tokens[9]->get_display_name() . '</option>' .
            '<option value="' . $tokens[10]->get_id() . '" >' . $tokens[10]->get_display_name() . '</option>' .
            '</select>'
```

Reply:
It'll look like this in code:

<img width="912" alt="image" src="https://user-images.githubusercontent.com/1288616/187376270-d59569f1-7c33-4515-b3c0-6b89b74ef539.png">

Reply:
Thank you so much for the guidance! I had indeed added a new line within the quoted string itself and that was causing failure. Updated it and it looks much more readable now!

Reply:
Thanks for fixing this. It looks much better now :)